From 0d5924ec4b89613910366c890305e46821a31f01 Mon Sep 17 00:00:00 2001
Message-Id: <0d5924ec4b89613910366c890305e46821a31f01.1649937143.git.geliang.tang@suse.com>
From: Sanjay R Mehta <sanju.mehta@amd.com>
Date: Fri, 17 Dec 2021 03:56:20 -0600
Subject: [PATCH] ntb_hw_amd: Add NTB PCI ID for new gen CPU
Git-commit: 0d5924ec4b89613910366c890305e46821a31f01
Patch-mainline: v5.17-rc1
References: bsn#170
Modified-by-SEL: No


Add NTB support for new generation of processor

Signed-off-by: Sanjay R Mehta <sanju.mehta@amd.com>
Signed-off-by: Jon Mason <jdmason@kudzu.us>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/ntb/hw/amd/ntb_hw_amd.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/ntb/hw/amd/ntb_hw_amd.c b/drivers/ntb/hw/amd/ntb_hw_amd.c
index 87847c380051..04550b1f984c 100644
--- a/drivers/ntb/hw/amd/ntb_hw_amd.c
+++ b/drivers/ntb/hw/amd/ntb_hw_amd.c
@@ -1321,6 +1321,8 @@ static const struct ntb_dev_data dev_data[] = {
 static const struct pci_device_id amd_ntb_pci_tbl[] = {
 	{ PCI_VDEVICE(AMD, 0x145b), (kernel_ulong_t)&dev_data[0] },
 	{ PCI_VDEVICE(AMD, 0x148b), (kernel_ulong_t)&dev_data[1] },
+	{ PCI_VDEVICE(AMD, 0x14c0), (kernel_ulong_t)&dev_data[1] },
+	{ PCI_VDEVICE(AMD, 0x14c3), (kernel_ulong_t)&dev_data[1] },
 	{ PCI_VDEVICE(HYGON, 0x145b), (kernel_ulong_t)&dev_data[0] },
 	{ 0, }
 };
-- 
2.34.1

